dev/docs/source/worksheet.rst

```diff
@@ -367,10 +367,12 @@ parameter is optional but when present is should be a valid
 :ref:`Format <format>` object.
 
 If required, it is also possible to specify the calculated result of the
-formula. This is occasionally necessary when working with non-Excel
-applications that don't calculate the result of the formula::
+formula (see discussion of formulas and the ``value`` parameter for the
+``write_formula()`` method above). However, using this parameter only writes
+a single value to the upper left cell in the result array and thus is largely
+unsuitable except for single-cell ranges::
```

Comment:
I agree that your proposed verbiage is better. If you're in no hurry, I'll try rebasing and resubmitting sometime in the next few days. If you get to a point where you'd rather go ahead and merge, that's fine too.